tests/test_acquire_command.py

```diff
@@ -4,7 +4,8 @@
 
 import pytest
 
-from acquire.acquire import (
+patch('ctypes.windll', create=True).start() # noqa
+from acquire.acquire import ( # noqa
```

Comment:
Are these `# noqa`s needed? I guess because the `patch()` is between imports? Is it not possible to patch after the imports?

If not maybe better to only disable the specific warning/error.

Reply:
Made it more specific, thanks :)